Incompatibility with Quarkus 2.8.0

After upgrading to Quarkus 2.8.0 the extension is broken because of:
```
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.quarkus.test.junit.QuarkusTestExtension.beforeEach(QuarkusTestExtension.java:410)
	...
Caused by: java.lang.NoSuchMethodError: 'java.lang.Object io.quarkus.test.junit.callback.QuarkusTestMethodContext.getOuterInstance()'
	at io.quarkiverse.test.junit.mockk.internal.SetMockkMockAsBeanMockCallback.beforeEach(SetMockkMockAsBeanMockCallback.kt:10)
	... 74 more
```

--- a/README.md
+++ b/README.md
@@ -18,7 +18,7 @@ First of all, you need to add the following dependency:
 <dependency>
     <groupId>io.quarkiverse.mockk</groupId>
     <artifactId>quarkus-junit5-mockk</artifactId>
-    <version>0.3.0</version>
+    <version>LATEST</version>
     <scope>test</scope>
 </dependency>
 ```

--- a/docs/modules/ROOT/pages/index.adoc
+++ b/docs/modules/ROOT/pages/index.adoc
@@ -13,7 +13,7 @@ In your `pom.xml` file, add:
 <dependency>
     <groupId>io.quarkiverse.mockk</groupId>
     <artifactId>quarkus-junit5-mockk</artifactId>
-    <version>0.3.0</version>
+    <version>LATEST</version>
     <scope>test</scope>
 </dependency>
 ----

--- a/integration-tests/pom.xml
+++ b/integration-tests/pom.xml
@@ -45,6 +45,7 @@
         <dependency>
             <groupId>org.assertj</groupId>
             <artifactId>assertj-core</artifactId>
+            <version>${assertj.version}</version>
             <scope>test</scope>
         </dependency>
         <dependency>

--- a/junit5-mockk/pom.xml
+++ b/junit5-mockk/pom.xml
@@ -45,6 +45,7 @@
         <dependency>
             <groupId>org.assertj</groupId>
             <artifactId>assertj-core</artifactId>
+            <version>${assertj.version}</version>
             <scope>test</scope>
         </dependency>
         <dependency>

--- a/junit5-mockk/src/main/kotlin/io/quarkiverse/test/junit/mockk/internal/SetMockkMockAsBeanMockCallback.kt
+++ b/junit5-mockk/src/main/kotlin/io/quarkiverse/test/junit/mockk/internal/SetMockkMockAsBeanMockCallback.kt
@@ -7,7 +7,7 @@ import io.quarkus.test.junit.callback.QuarkusTestMethodContext
 class SetMockkMockAsBeanMockCallback: QuarkusTestBeforeEachCallback {
 
     override fun beforeEach(context: QuarkusTestMethodContext?) {
-        context?.outerInstance?.let {
+        context?.outerInstances?.map {
             MocksTracker.getMocks(it).forEach(::installMock)
         }
         context?.let {MocksTracker.getMocks(it.testInstance).forEach(::installMock)}

--- a/pom.xml
+++ b/pom.xml
@@ -17,8 +17,10 @@
     <properties>
         <kotlin.version>1.6.20</kotlin.version>
         <mockk.version>1.12.3</mockk.version>
-        <quarkus.version>2.7.5.Final</quarkus.version>
+
+        <quarkus.version>2.8.0.Final</quarkus.version>
         <dokka.version>1.6.20</dokka.version>
+        <assertj.version>3.22.0</assertj.version>
 
         <!-- This plugin is not compatible with kotlin -->
         <maven.javadoc.skip>true</maven.javadoc.skip>

